internal/api/controllers/public/conversions.go

```diff
@@ -57,9 +57,6 @@ func dbRuntoApiRun(r *dbModel.Run, fields []string) *Run {
 		switch field {
 		case fieldId:
 			run.Id = (*RunId)(convertUuid(r.ID))
-		case fieldAccount:
```

Comment:
in addition, the account field should be removed from the list of allowed fields https://github.com/RedHatInsights/playbook-dispatcher/blob/5fe8a813104459d0ab618b043fe27f8963e407aa/internal/api/controllers/public/conversions.go#L33